packages/keycodes/src/index.js

```diff
@@ -33,16 +34,7 @@ export const CTRL = 'ctrl';
 export const COMMAND = 'meta';
 export const SHIFT = 'shift';
 
-/**
- * Return true if platform is MacOS.
- *
- * @param {Object} _window   window object by default; used for DI testing.
- *
- * @return {boolean}         True if MacOS; false otherwise.
- */
-export function isMacOS( _window = window ) {
```

Comment:
It looks like this method targets desktop Apple devices:
https://stackoverflow.com/questions/19877924/what-is-the-list-of-possible-values-for-navigator-platform-as-of-today

Reply:
But window will not exist on a RN context correct?

Reply:
Yes, you should omit it in RN override. I think it's done this way to be able to mock it in unit tests.